support-frontend/app/actions/CustomActionBuilders.scala

```diff
@@ -39,11 +39,11 @@ class CustomActionBuilders(
   private val idSkipValidationReturn: (String, String) = "skipValidationReturn" -> "true"
 
   private def idWebAppRegisterUrl(path: String, clientId: String, idWebAppRegisterPath: String): String =
-    idWebAppUrl.value / idWebAppRegisterPath ?
+    (idWebAppUrl.value / idWebAppRegisterPath ?
```

Comment:
check that there are no `/` in the `idWebAppRegisterPath`, as I discovered these are now escaped automatically (which is probably correct, but it's a change in behaviour)
https://github.com/guardian/membership-common/pull/608/files#diff-4ab30be5dbdbc76f24180331be757b19R8

Reply:
Oh that's nasty! Good spot.
In this case, it's only ever called with the string `"signin"`:
https://github.com/guardian/support-frontend/blob/f00be2f4a2a495fb87025e34476125357e71984e/support-frontend/app/actions/CustomActionBuilders.scala#L49
Do you think it's safe to leave it as it is?

Reply:
in that case fine, if it works it works.  Could be worth just inlining the "signin" as no point having a parameter if it's never changed

Reply:
good point!